src/app/appointments/AppointmentsList.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+// import { Appointment } from "@/app/data/appointments-data";
+import * as React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+
+import { Box, Tooltip } from "@mui/material";
+import Container from "@mui/material/Container";
+import Sheet from "@mui/joy/Sheet";
+import EditCalendar from "@mui/icons-material/EditCalendar";
+import Delete from "@mui/icons-material/Delete"; // Uncomment if you have a delete icon
+// import { appointments } from "@/app/data/appointments-data";
```

Comment:
```suggestion
import * as React from "react";
import Table from "@mui/material/Table";
import TableBody from "@mui/material/TableBody";
import TableCell from "@mui/material/TableCell";

import TableHead from "@mui/material/TableHead";
import TableRow from "@mui/material/TableRow";

import { Box, Tooltip } from "@mui/material";
import Container from "@mui/material/Container";
import Sheet from "@mui/joy/Sheet";
import EditCalendar from "@mui/icons-material/EditCalendar";
import Delete from "@mui/icons-material/Delete"; // Uncomment if you have a delete icon
```

![image](https://github.com/user-attachments/assets/79dd6059-4b00-44c9-bf11-d1f8cd6986ff)